src/Nuget.PackageIndex.VisualStudio/Analyzers/ProjectTargetFrameworkProviderExporter.cs

```diff
@@ -29,7 +29,7 @@ internal class ProjectTargetFrameworkProviderExporter : IProjectTargetFrameworkP
         /// <summary>
         /// Stores file and it's corresponding unique project name
         /// </summary>
-        private Dictionary<string, string> FilesCache = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+        private Dictionary<string, string> FilesCache = new Dictionary<string, string>(StringComparer.InvariantCultureIgnoreCase);
```

Comment:
This should be ordinal , I probably switched form other repo where we used Invariant all the time. I will update to Ordinal in all places in this repo.